types/types.cc

```diff
@@ -1609,6 +1631,124 @@ static void validate_aux(const tuple_type_impl& t, View v) {
     }
 }
 
+sstring vector_type_impl::make_name(data_type type, size_t dimension) {
+    // To keep format compatibility with Origin we never wrap
+    // vector name into
+    // "org.apache.cassandra.db.marshal.FrozenType(...)".
+    // Even when the vector is frozen.
+    // For more details see #4087
```

Comment:
Should we refer to this issue? It was created long before the vector datatype was implemented in Cassandra.

Reply:
After adjusting the implementation to match Apache Cassandra, reference to this issue has been removed as it became irrelevant.